### ISSUE TITLE
Issue with Load in jupyter notebook.

### ISSUE THREAD

**Issue by SimonKato** [SiqiHuang18/Ischemic-Stroke-Lesion-Segmentation#1]
In the Load block the following lines are found: 
train_csv_path = '/content/drive/My Drive/data/train_dataloader_final.csv'
val_csv_path = '/content/drive/My Drive/data/validate_dataloader_final.csv'

There is no reference to how train_dataloader_final.csv is or has been created. I am also using the ISLES2018 training data. The train_1.csv, train_2.csv, ..., train_5.csv is in the cross-validation folder. Feeding the path to the csv folder folder with the line
train_csv_path = 'cross-validation/train_1.csv'
val_csv_path = 'cross-validation/validate_1.csv'
This results in the issue with the following block of code:

sample = next(iter(dataloader['train']))
sample['x'][0].size()

The intended output is torch.Size([1, 256, 256]), but I get 
![typeerror](https://user-images.githubusercontent.com/38088338/99178877-ae62aa00-26e5-11eb-81f2-05a62f0611df.JPG)

I have attempted using the Dataloader to merge five channels. The afromentioned code block then yields
torch.Size([5,256,256]), but then an issue in the training loop of Unet Attention:
![unet attention](https://user-images.githubusercontent.com/38088338/99178915-3c3e9500-26e6-11eb-827d-b72b70dae5ff.JPG)

In conclusion, is it possible to release the file train_dataloader_final.csv and validate_dataloader_final.csv and train_dataloader_final_v3.csv and validate_dataloader_final_v3. Moreover, I am attempting to train the model with the ISLES2018 data, considering this is the same model that has already been trained, could you release the model?
Thank You

**Comment by cosinec** [comment]
I have the same question. Did you solve it?

**Comment by SimonKato** [comment]
> I have the same question. Did you solve it?

Yes, the issue I was having was that the docker build file was outdated. I believe that the link to download miniconda needed to be replaced with a different mirror.
